cpp/src/io/parquet/reader_impl_preprocess.cu

```diff
@@ -1076,213 +361,6 @@ void reader::impl::read_compressed_data()
                "Encountered page_offsets / num_columns mismatch");
 }
 
-namespace {
-
-struct cumulative_row_info {
```

Comment:
Removed this struct all together as it was not being used anywhere. There is another struct called `cumulative_page_info` (defined in `reader_impl_chunking_utils.cuh`) which looks identical and is being actually used